bootstrap/rackspace/config-default.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+export TF_VAR_region=${TF_VAR_region:-LON}
+export TF_VAR_auth_url=${TF_VAR_auth_url:-https://identity.api.rackspacecloud.com/v2.0}
+export TF_VAR_mesos_slaves=${TF_VAR_mesos_slaves:-1}
```

Comment:
dont think this var is needed (should be set to a default in the terraform config)